app/views/curation_concerns/base/_metadata.html.erb

```diff
@@ -1,12 +1,18 @@
 <h2><%= t('.header') %></h2>
-<table class="table table-striped <%= dom_class(presenter) %> attributes" <%= presenter.microdata_type_to_html %>>
-  <thead>
-    <tr><th>Attribute Name</th><th>Values</th></tr>
-  </thead>
-  <tbody>
-    <%= render 'attribute_rows', presenter: presenter %>
-    <%= presenter.attribute_to_html(:embargo_release_date) %>
-    <%= presenter.attribute_to_html(:lease_expiration_date) %>
-    <%= presenter.attribute_to_html(:rights) %>
-  </tbody>
-</table>
+<% attributes ||= [ :embargo_release_date, :lease_expiration_date, :rights] %>
+<% presentations = attributes.map { |attribute| presenter.attribute_to_html(attribute) }.reject { |x| x.empty? } %>
```

Comment:
@jcoyne OK, practical questions.  The presenter does not necessarily fully know which of its attributes are metadata and which are just... regular data or something else.  We delegate these to the solr_document:
```
:based_near
:related_url
:depositor
:identifier
:resource_type
:tag
```

`CurationConcerns::WorkShowPresenter` delegates what a comment calls "# Metadata Methods":
```
:title
:date_created
:date_modified
:date_uploaded
:description
:creator
:contributor
:subject
:publisher
:language
:embargo_release_date
:lease_expiration_date
:rights
```

But that set may change underneath us (i.e., minor version can add to it) and there is no enumeration of that set for us to reference downstream.  Also, a check like `empty?` should be cheap, right?  It shouldn't have to interrogate 20 methods underneath it.  Lastly, right here we only care about 3 fields.  Some metadata fields could well be populated, but not the 3 we care about.  We are worried about collapsing the display of 3 possible fields as part of the display logic.  I'm fine keeping that in the template.